front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Properties.java

```diff
@@ -37,6 +43,10 @@ public void setRootFolder(String rootFolder) {
     this.rootFolder = rootFolder;
   }
 
+  public Integer getMaxKeys() { return maxKeys; }
```

Comment:
We generally add newlines even to 1-liners

Reply:
Oops, was collapsed in the IDE. I'll update the PR.